include/keymap.h

```diff
@@ -189,13 +189,17 @@ class KeyMap : public ConfigActionHandler {
 	void dump_config(std::vector<std::string>& config_output) override;
 	std::vector<KeyMapDesc> get_keymap_descriptions(std::string context);
 
+	std::vector<MacroCmd> parse_operation_sequence(const std::string line);
```

Comment:
Why is the argument here passed by value? It doesn't look like a typo since you have the same signature in the cpp file, but I miss the point of this.

Reply:
I'm not completely sure how this happened, it is definitely not what I intended.
Fixed.